insns.def

```diff
@@ -253,6 +253,28 @@ setclassvariable
     vm_setclassvariable(GET_ISEQ(), GET_CFP(), id,  val, ic);
 }
 
+DEFINE_INSN
+opt_getconstant_path
+(IC ic)
+()
+(VALUE val)
+// attr bool leaf = false; /* may autoload */
+{
+    const ID *segments = ic->segments;
+    struct iseq_inline_constant_cache_entry *ice = ic->entry;
+    if (ice && vm_ic_hit_p(ice, GET_EP())) {
+        val = ice->value;
+
+        VM_ASSERT(val == vm_get_ev_const_chain(ec, segments));
+    } else {
+        ruby_vm_constant_cache_misses++;
+        val = vm_get_ev_const_chain(ec, segments);
+        vm_ic_track_const_chain(GET_CFP(), ic, segments);
+        // INSN_ATTR(width) == 2
+        vm_ic_update(GET_ISEQ(), ic, val, GET_EP(), GET_PC() - 2);
```

Comment:
```suggestion
        // Because leaf=false, we need to undo the PC increment to get the address to this instruction
        vm_ic_update(GET_ISEQ(), ic, val, GET_EP(), GET_PC() - attr_width_opt_getconstant_path());
```

Reply:
👍 I was trying `ATTR_WIDTH` and not having success. 🤞 this works

Reply:
ah, yeah. This doesn't work under MJIT :(